src/+otp/+ascherlineardae/AscherLinearDAEProblem.m

```diff
@@ -0,0 +1,36 @@
+classdef AscherLinearDAEProblem < otp.Problem
+  %Index-1 DAE problem
+  % Problem description and fortran implementation can be found in
+  % Ascher, Uri. "On symmetric schemes and differential-algebraic equations." 
+  % SIAM journal on scientific and statistical computing 10.5 (1989): 937-949.
+
+    methods
+        function obj = AscherLinearDAEProblem(timeSpan, y0, parameters)
+            obj@otp.Problem('Ascher Linear DAE', 2, timeSpan, y0, parameters);
+        end
+    end
+    
+    methods (Access = protected)
+        function onSettingsChanged(obj)
+            beta = obj.Parameters.beta;
+            
+            obj.Rhs = otp.Rhs(@(t, y) otp.ascherlineardae.f(t, y, beta), ...
+                otp.Rhs.FieldNames.Jacobian, @(t, y) otp.ascherlineardae.jac(t, y, beta), ...
+                otp.Rhs.FieldNames.MassMatrix, @(t) otp.ascherlineardae.mass(t,[], beta));     
```

Comment:
Let's add `MStateDependence` property to RHS

Reply:
Should it be an Rhs property? Do we support other ones line JPattern?

Reply:
Yes.  Don't think JPattern is supported yet, but can be added easily